tests/test_wikipedia_home_page.py

```diff
@@ -9,3 +10,14 @@ def test_wikipedia_home_page(pages):
     with allure.step("Verify there is a link to 'anyone can edit' section and you can click it"):
         assert pages.wiki_home.anyone_can_edit.text == "anyone can edit"
         pages.wiki_home.anyone_can_edit.click()
+
+
+@allure.title("Search for an article in main page")
+def test_wikipedia_search_field(pages):
+    with allure.step("insert text in search box"):
+        time.sleep(1)
```

Comment:
You can do it in another pull request, but please add a retry here instead of sleep. You can use the retry that we've implemented before. The retry usage would look something like this:
```
retry_until_not_fails(pages.wiki_home.search_bar.send_keys, "Rihanna")
```
Retry will do sleep in the background if needed. This will prevent you from **always** sleeping, you will just do a 1 or more second sleep when needed. The retry will try to do `send_keys` and if it fails it will sleep and then try it again. Such approach is more reliable and stable :)